lib/LicenseTermsList/tests/interactor.js

```diff
@@ -0,0 +1,22 @@
+import {
+  interactor,
+  text,
+  isPresent,
+  count,
+  collection
+} from '@bigtest/interactor';
+
+@interactor class LicenseTermInteractor {
+  isLabelPresent = isPresent('[data-test-term-label]');
+  isNamePresent = isPresent('[data-test-term-name]');
+  label = text('[data-test-term-label]');
+  name = text('[data-test-term-name]');
```

Comment:
So this should be changed if reverting that change to the dataset props. More accurately and sensibly now, it'd have a `value` prop for the interactor instead.